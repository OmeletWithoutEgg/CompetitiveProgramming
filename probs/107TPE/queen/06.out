17005267.49 26632660.93
