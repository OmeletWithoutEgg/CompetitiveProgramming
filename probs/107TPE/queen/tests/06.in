1044
124925396 58797169
124713347 51273719
-78603730 -15265360
-35627731 16806698
-50800599 59182527
-37812046 35317813
-50799693 -26954545
67572787 70797195
112984461 71923130
-43791257 50792855
-55764354 -29761671
-46842573 55225571
79395968 35331576
77746655 35748508
-75245949 -21074643
81882004 76000730
-48989584 -25359601
-84864761 21287672
91430733 33986887
68952572 71592351
-53439487 60941621
-79788083 -12607902
-42417035 -16036156
82392381 76090095
96890122 76539066
-73563781 55512035
-84814229 22082109
122978506 63267966
76978770 74853632
-35658369 13688297
125037226 52493622
-50956107 59304150
69233228 71743718
-84921390 20215947
-77237051 50120816
-71804233 57437391
75831769 74503700
119631728 67291755
-59576851 -30541876
-37718332 -2693235
-84983369 18547141
-70211906 -26655924
-40943117 -12822671
61449371 45054205
-59411327 62758662
-38832530 39117804
-74090863 -22633908
122528261 46867174
57205113 55255708
-35625344 15870290
58571093 61415649
-37344761 33227162
58455636 61161335
-37307365 33046709
69296545 39040746
78130591 35645808
-83983799 29473270
-52678980 -28272081
-69493621 59451272
-52577939 -28209480
-58364598 -30416378
124101357 61164736
-83462850 32410343
65285321 69260422
-74080787 54873960
-82368905 37141711
116222778 70003842
92793442 76810320
68055716 71085553
-46975031 55387498
124475991 60241476
57410844 57764536
89497566 76802178
58219364 50219095
61756833 44711601
-73938216 -22826142
-36996021 777708
-37215990 -368935
98638737 76325870
62810635 67181494
-74925123 53754182
-67872192 60554823
-41223451 -13480968
-48640368 57239314
94750163 34098101
-64934570 61968596
-77168051 -18015690
-36598424 29089280
-69753844 -27023998
-38682637 -6383014
-54818841 61617133
78935268 75376417
57234355 56309540
81233228 34938847
89737840 76809552
-73438406 55661581
-43047811 -17246137
-83491888 -35007
-35987227 24078693
-81642283 -7439583
-37216590 32599208
95763945 76645133
92070651 33992454
124559547 50809441
-79494717 45531688
-80434372 43197682
-64468423 -29900131
-70777265 -26172785
-82666309 -3761372
116807359 69598708
-76035460 52121319
-40906645 -12735206
57643641 58838266
76884401 35992067
69762036 38798073
66156990 69882169
67103973 40311998
-65258990 61847734
113786434 71493223
82986266 34630795
-82699812 35853522
118102240 68620917
118176313 68561261
104129832 35568466
-81972306 38565623
57396516 53136521
118324917 42377900
-77750524 49181579
-82316957 -5107896
-46364993 54622580
-56345843 -29954427
-48178388 -24531236
81318890 75895849
-48771820 -25144456
73907023 36979829
111145753 38004321
68252072 39618534
-39485355 41171475
-48662493 -25034468
114615941 71018995
60833305 45788969
101902772 75763747
113594998 39219848
-35658397 13687276
125269735 56917044
-83145370 33938184
77977325 75131769
70765202 38303796
87050556 34152276
-35787968 10760295
-60064862 62787827
82396766 34727317
-37228419 -431044
-62082109 62671500
-83104450 -1896964
-57842893 62554752
124351101 50249310
92340663 33997071
-69837670 59185443
108112518 36783191
-65327210 61821232
-82302218 -5162382
81989443 34798167
66067102 40997899
-52542891 60414820
57214779 55941861
-51794692 -27692952
89075608 76786638
113357023 71726761
-55024503 -29476958
-37188405 -230134
-36723172 2349364
-78620010 47458709
114777274 70923041
100753542 34831385
-36045257 7539310
123939810 61517161
-47429648 55926276
125101313 52803682
-37333015 -943453
-65068055 -29692632
75236853 74308818
-70865935 58321322
81079284 34968958
120365839 66568466
-82117821 -5829622
-43648900 -18321289
98085144 34418328
-57500779 -30257718
-53279998 -28625793
-36645571 2833936
123881725 61638603
64241862 68447044
-56245319 62150305
-61617099 62726009
-43058178 49492567
-46697540 -22818387
-84116640 28601968
109525817 73501293
94704730 76722961
-75986225 52197874
64151920 68372992
57432829 57886292
92740646 34006386
-35627930 15395679
125154010 57728149
-65787080 61632648
-41304888 -13667709
-60613363 62787066
59195419 62624826
-56831200 62321957
-78001317 -16475227
-56349770 62182903
124666515 51126781
77018894 35952882
92348268 33997220
-59294129 -30522704
99144064 76253058
-50491635 -26706607
92054591 76825935
-46017245 54164035
62514324 43924878
101810465 75782697
61352985 65653440
116431923 40956881
-83455453 32447819
-40187183 -10916193
-80710497 -10222531
87571172 34113896
80704549 35044365
58230581 60627222
59432349 47789774
-56394663 -29969381
-76114265 -19770672
74520392 74061468
-78374596 47963026
124515716 50685838
57214961 55946441
-35674055 19053925
-38016267 -3923676
89644111 34011354
125346894 55854522
113789358 71491606
125338035 56070111
-54431582 -29216371
-84065696 3283580
-84664836 8259595
99706425 34652146
96652965 34254741
-80988508 41663940
-82950161 34806187
-62137591 62663884
122652038 47048423
-47308872 55785671
-83625257 31561488
-64002649 62268734
124807783 59230311
62057688 66428213
110670450 73024429
-56162189 62123739
-55867829 -29797955
75872383 36301492
73281565 73599557
121460855 45465196
117807755 41964288
123292033 48069667
63624374 42893469
-41707867 -14563686
121308692 65533405
-37375162 33372259
81327961 34920566
-80816081 42155384
113589838 71601120
118119642 68606940
-63453873 62413229
116096744 70088518
79111827 35398954
77104882 35928063
109129846 37161405
-56376181 -29963742
66023542 41028078
-75816542 -20231392
59362769 62912428
93704044 34040984
120999343 65887983
-49892247 58424348
-36361982 4795795
62847088 67216262
-80261474 43650604
82391079 34728283
102921633 35275870
-81720604 -7184463
-82926411 34908560
99393912 76215171
61907531 66269276
101842468 75776148
78065311 75155133
-59846547 -30554456
-53291002 -28631974
108217146 36820718
57231761 54548260
107361815 74295321
57403162 53097693
-84237490 27748090
-48580168 -24950764
114477747 71100198
-83338450 -800731
85281717 34321113
-51432587 59662344
-35644973 14237168
73291006 37214910
-39431147 -8782457
91915292 33990408
98983871 76276691
87441501 76695179
69796385 38780510
125270403 56911085
-71477763 57755642
-78736110 -14987600
-53995351 61233186
-81653568 39630320
-36981838 31372283
108531238 36935240
-50820816 59198472
89910435 76814190
76726208 74779427
108431785 36898674
-59147370 -30510334
-56224650 -29916484
88874042 76778048
-47512993 -23794988
-37254628 32788404
-65156600 -29659534
125308167 54295187
60910413 45693117
-36954428 1005588
117186837 69324212
-84952202 12735751
122478995 64021868
125234559 53615026
-64085395 62244884
-84424200 5948387
59070491 48416765
62386543 66765790
57619253 58742344
61826620 44635837
-36757875 30088669
120182314 66755370
118157271 42241518
-75033658 -21375620
-78668636 -15129733
63365207 43122976
-53615863 -28809734
66647319 70211533
-84136353 28467001
-51575593 -27538384
78552410 35536966
125326884 56267653
67359320 70666071
83175552 76216860
-53111589 -28529878
-50911924 -27042566
-39291626 -8359258
86270421 34219380
-71828078 57413690
101134581 75915915
124591062 50900508
88190616 34074860
-39665172 41697767
63920149 68179035
-43165835 -17462992
87838584 34096163
-40651439 44338158
92926772 76806427
-42443571 -16088862
-79767607 44884679
82715709 34674217
57222644 56113072
-84235857 4467193
62785978 43660264
111344367 38094607
-81136324 -9004357
-84961307 19245340
87205482 34140321
57209686 55026056
-77097607 -18139425
-56862670 -30103141
115304908 70600269
-78624779 47448760
81294357 75891128
103533603 75398226
-47220202 55681286
124880469 58969031
-85007989 14856036
67796356 39886064
116889723 41278232
65930928 41092626
104093933 35559291
-82182375 37826269
-81660826 -7379517
-46284655 54518130
-63101776 62493486
-83529148 32069993
81724869 34846027
78009407 35677848
57267919 54100009
-49746495 -26067652
57213071 55896778
115864788 70241986
-40275944 43378104
-77408677 -17585637
-77754448 -16946922
-36028790 24519696
58022752 60076253
-56363085 62186997
-38292830 37215604
117204821 41507190
100706206 34822795
-74207358 -22485106
-77192738 -17972099
87007442 76662474
58684622 61654808
-48557843 57155201
68105812 39703331
-47071088 -23276246
-76135885 51963909
-43095916 -17334873
64920296 41833122
106118241 74693490
125303593 56576885
119364178 67540283
-43094176 -17331672
-76312150 -19456226
-49130500 -25496060
-39169398 -7980032
57447805 52852283
62490173 66869335
125085346 58095129
-73422094 55680892
100323314 76063144
67154318 40280201
-37914262 -3512910
122438149 64080115
-39596597 41498884
59624547 63338625
-81818566 -6860036
-71162153 58052381
-35674863 19078018
91903362 33990269
-79591778 45304250
-81257574 40869014
-40866107 -12637484
-54746075 -29358216
78777957 35480460
93620378 34037296
-35626525 15596400
64786068 41936787
60240921 64249375
-43760690 -18513453
-42814390 49035861
118812787 42788173
-70302758 58807724
57512451 52534293
-82629602 36135403
-48491084 57086597
105979367 74735483
100719746 34825247
-70821434 58360936
-51142315 59446713
115405847 40281249
105327918 35892107
-85016953 16099202
103363069 75439217
-41561865 -14244330
-52773859 60557303
-67621374 60704286
-61012724 -30544756
-65861266 61600596
118349118 42397777
61661885 66002316
-35718755 12050437
-37890292 35642123
-81194829 -8830315
-58699220 62689584
99269530 34583968
67784271 70924863
120560656 44452912
58956059 48628683
-82593418 -4051531
57266435 54115340
-84747442 22990709
-66929059 61088339
92884410 76807700
105131807 74981376
-50361054 -26598682
-82747985 -3429972
123931815 61534033
-37603290 -2191373
123371135 62610658
-64469166 62127159
108526200 36933381
-43803456 -18586346
58041627 50689059
-84995382 18064181
-84059242 3240984
-84576825 7340620
101117180 34898940
-64328767 62171588
-46631459 54962760
-84929293 12184280
-77116950 -18105545
113747469 71514741
-81495285 40135902
-36343183 4938916
59207284 62645643
123980733 49388160
-43345674 50015206
95042321 76700448
123852828 61698065
125352445 55630261
119314099 67585979
102073980 75728113
110611703 73049932
-82065945 -6012721
-83277217 -1095439
125347180 55845789
-58287469 -30404563
67586176 70805343
96316476 76596117
-73780080 -23022052
88466182 76758356
98151525 34426878
-84649429 24136401
-55945126 62051764
-44143808 51381751
-60866877 62778933
95483054 34150324
-35636076 17510914
-82970995 -2488560
59695210 47368935
-48132231 56709119
-35857996 22510117
-48667390 -25039423
111131470 37997880
-37209168 32562001
71252319 38077142
81002240 75833940
58610467 61499754
-36020314 24431724
111173783 38016980
-78654296 47387058
121584453 65203258
-47696792 56230917
-55731912 61977384
66728294 40553549
86910040 76654646
125029774 52459719
-55229683 -29560385
-84160226 28301400
123329935 48135342
60781903 64964619
-69425424 59502613
102743590 75582628
-49877285 58411163
-35775317 10971631
125222433 53526518
58195793 60539353
67752279 39912466
125135462 52985314
57782979 59340547
61850642 66208235
-37706246 34868526
109956584 37491444
123040690 63168098
124857705 59052937
116816044 69592530
124979212 58578726
-53807747 61137735
123951393 49325528
-78786230 47108529
81701352 75967806
-85016861 15984974
63481453 67798914
101674119 35007803
76214992 74624335
-71876305 57365559
92052323 33992189
-74912750 53771320
89798179 34006921
59533422 63193308
-41151309 -13313880
124865637 59023928
-36091333 25141527
-37986076 -3803178
-59232411 62744970
60030495 46866804
-81725305 39396298
-84230953 27795990
-67291752 -28665053
-83092146 -1952463
-63927180 -30062747
82241236 34753967
-40325636 -11280877
118592474 68217792
106736522 36317588
-45374514 53271051
80069347 75639064
71821512 72995405
57227894 54611001
-35983208 24034702
-83758621 1404191
107224297 36476806
-63097384 -30267159
99123985 34562109
-64966199 61957187
123145642 62996106
-39097549 39980524
123668001 62064675
-71233323 57986392
117056756 69419370
112609531 72115104
125008850 58451576
75285200 74325007
104541155 35675738
73311365 73611197
-59576106 -30541834
-35844243 9907934
-49076667 57671451
59432491 63028618
96663151 76562392
64690814 42011161
-46317444 -22333601
-41309200 45904809
-55909578 62039617
82767243 34665831
97499338 76471547
63975229 68225544
-47595510 -23889176
116475957 69830928
-62504033 62607613
-40932904 45025490
63810791 68085906
124849714 51736232
78168550 75182313
60374393 46385855
118911698 42873905
-69620907 59354254
-48624045 -24995470
-71349396 -25650349
110148398 73247103
-36883385 30823320
61064996 65313694
58425577 61093105
-84647531 8070332
85993760 34245958
115213943 40161240
80428258 75716265
59006013 62282881
-43417502 -17915906
119090542 43031217
124960884 52163343
-66094085 61497050
74837725 74172760
123435441 48321782
79065552 35410099
-43833833 -18637914
123267956 48028292
77941134 75122107
-39650578 41655624
-40724160 44518291
59813713 63630817
-77843329 -16778639
64225543 42384499
-58310074 -30408073
123124209 47786564
-65013769 61939877
64300708 42323022
-36744057 2222101
101074359 75927316
-82176009 -5621883
-50277362 58755881
124745437 59440540
-74379249 54489451
-40480207 43906720
-67480849 -28558341
-73967126 -22789972
-46287558 -22294654
-38955836 -7297227
-35662732 13534689
-77497534 49651143
63200642 67546062
82842947 76164546
72500285 37533480
57294790 53848367
-35668678 13339043
-36700155 2491127
109867210 37454747
-73491851 55598076
116703509 69672212
-60194471 -30562458
115596852 70415544
123114908 63046932
-44534278 -19781015
-84310160 27200757
85699127 76542284
63899818 68161800
125230516 57233123
103239111 75468601
58622038 49293926
102959978 35284650
-56307663 -29942600
-82484354 -4476325
113775287 39318771
-80906189 41900245
61125886 65386765
-48787248 -25159871
-44830221 -20238701
92570208 76816102
-49426443 58002950
83004753 76190275
101820303 35037471
125262535 53838280
-83457937 -207979
-36478237 28276916
123140904 63003969
-48416407 -24781991
-84979595 13547162
-65985436 61545931
-84984824 13733389
120681695 44581790
-58200344 -30390664
60124236 64085629
-65950496 61561442
67554436 40032145
-52426041 -28113653
75836232 74505127
67026208 40361376
85431304 76513643
68568259 71379589
-44430265 -19616682
-84757890 9370465
-35885123 22870385
61364852 65667129
-51896133 -27763008
-84208075 27961998
-40285754 43403852
-57105863 -30165821
89695240 76808323
-55766018 61989528
-68239493 -28098597
-44240111 -19311465
82859069 76167133
-47141263 -23360259
58387550 61005507
-78960252 46734169
-40082018 -10634155
-55034114 -29480943
-39871175 -10055004
123628491 62140194
125253036 53759067
63500900 43002016
-53192632 60803611
58100472 60289991
64257248 68459645
91243651 33986687
89939205 34003247
-45685513 53710691
-67252396 -28686871
69862678 38746745
68708989 71458249
118652961 68166655
-41442714 46206587
-43869128 -18697617
-59784619 -30552061
-37598625 34397941
-36939872 31141020
110114948 37557087
124071022 49585482
70652548 72460731
114819889 70897485
58296978 50027456
-61511642 62736057
-48572678 -24943111
115735953 70325930
95692540 34167112
119985538 66951126
-61423130 62743831
-78189635 48333487
114698643 70969964
-60240842 -30562825
-44006269 -18927426
84796759 34377907
-73379137 55731587
92345936 76820983
-63790476 62327414
-84995867 18042133
-58505193 62663999
-69276778 59612986
-64042731 -30029982
125349261 55776078
-37311765 33068057
-84381652 5594472
59441513 47774652
-36131987 6704349
-84977646 13481103
-48258953 -24616841
85507321 34296245
59154183 62551944
82201314 76057269
67871693 70977000
-48118724 -24467354
-84514209 25483962
78828453 35467970
72123205 73125829
-78029799 48647202
-84597309 7545415
-51850303 -27731475
116547974 41037117
95988600 34192254
90215540 76821041
60123170 64084116
67551675 40033829
-78418895 47873087
86890981 76653094
124428582 60368441
-77563810 49529430
125313550 54362205
57565226 58519056
122420559 46713102
-45630203 53633552
61796344 66149539
-80997782 41637121
118473260 42500498
63253570 67594317
-81821169 -6851331
88629452 34051546
92453387 76818760
-65565279 -29498502
-60210515 -30562604
-69302126 59594313
-50980216 -27095529
-42262148 47952455
-58280040 62630668
90468228 76825413
124606393 50945515
-36477890 3952778
-38755495 38858825
57479223 52693012
122395240 46677353
97241947 76500948
-63367835 62433736
-61755209 62711557
-82501461 36637679
-40275738 -11150298
122740204 47180409
82734593 76147019
108833722 73769945
-44569021 -19835499
-35872133 9526894
-67846690 60570273
-75527545 52892122
69959110 72120219
106540243 36255278
-67668447 60676658
-35799656 10572296
115741142 70322566
-35949707 23657778
-84942608 12494490
-61838554 62702126
-81584483 39852778
59526493 63182131
116686383 41133888
122039993 64624884
-82305150 37378824
-37581953 -2096485
95860632 76636975
-35691490 19536259
95794723 34175602
57204966 55269152
-62272031 62644445
100788260 75980441
-37635347 -2332849
-75149436 -21212311
-48512485 57108644
115179764 40140064
-58403596 -30422176
-60443626 62789762
73569774 37107139
-84327064 27069273
125267340 56938237
-82329906 -5059884
71087263 72665157
-42265083 47958401
-36175934 25917847
66198065 69910298
104157983 35575682
-35639986 14488250
83970867 34485206
-49978732 58500114
119656948 67267936
-81835046 -6804860
64539326 68687301
-39318438 -8441369
70938629 38222133
-72723843 -24249519
124879386 58973071
-76082366 52048006
89090532 76787244
95692355 76651060
125202336 53388407
-71450054 -25554854
70682185 38343272
58702166 61690883
-69438936 59492476
-84039262 3110085
-74226043 -22461068
-77929982 48840176
72432153 37561843
-84849714 21536590
84214428 76365989
-60450153 -30562432
-51633447 59806900
113477149 71662220
-36618707 3006244
-53528390 -28762766
123954785 61485423
-60874883 62778596
124980588 52245216
59177241 48225356
111843680 38327723
106727798 74503359
102791465 35246307
-84898938 20671427
-71354536 57872766
-40866849 44866545
-84333423 27019379
-38800301 39009924
-52192996 -27962592
124188038 49852492
-36397766 27698725
73556522 37112210
119361215 43275163
105289150 35881096
-47326662 55806495
-51558543 -27526168
118662959 68158170
64658700 42036389
105811398 36032531
57816351 59451310
-77099533 -18136055
-70759556 -26188412
62141407 66515492
-43077805 49528789
62706029 43737258
81013451 75836168
-37787512 35214757
91995861 76826744
-68433023 60200351
113736353 71520868
-77341444 -17706963
77075903 35936407
75201762 74297029
113427584 71688923
120531766 44422452
96525010 76576106
-58194073 62616908
-72401190 -24597589
96123383 34204251
94460000 76737946
-35627516 16778746
121127099 45074690
124854336 59065186
-49376439 -25729092
-35928078 23403791
72033950 37730601
-48121843 -24470704
-44831411 -20240512
90507638 33992169
71314592 38048769
112083676 38442978
86516608 76621201
-41063023 45334500
-35697742 12533920
-39136352 -7876090
-44623724 52148141
-77430439 -17546169
-68027056 -28232490
104599259 75126945
114131074 71299990
-36450516 28081110
-47172159 -23397049
124375918 60505566
76362992 74669915
-80335864 43457130
-69761108 -27018321
72180884 37667733
57742158 59200504
67469506 70734040
74468011 74042828
-35812530 10372446
-58796866 62701363
-42649188 -16491773
99281892 34585844
-84545429 25191247
-72168710 57068187
-44166772 -19192065
-72129916 57108193
57205646 55214430
-36261455 26642143
68543520 71365671
115947152 40630318
-63253952 -30232722
118558732 68246181
67139463 40289569
-36074824 7245487
-35709514 19971717
58818575 61924708
-45821721 53898749
87338019 76687704
-41082470 -13152958
-76340945 51637176
-59084819 62731826
-43998468 51141712
111904076 72461628
109426927 37277595
57639647 51995411
-80728024 -10173991
103631471 75374403
100070621 34711936
-70277046 -26601898
66319402 40825347
-38444096 -5543189
-41003557 -12966676
78845267 35463824
125249107 53727397
108104236 74037923
79185180 35381385
118485042 42510315
58936937 48664820
-36069543 7297230
97812581 76434031
-67674569 60673050
118606449 42612151
-77770701 -16916277
