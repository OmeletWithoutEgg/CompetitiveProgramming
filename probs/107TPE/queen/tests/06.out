1429077.26 -18434743.11
